db/migrate/20170221225907_create_items.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :items do |t|
       t.string :title
       t.string :description
-      t.decimal :price, :precision => 2
+      t.float :price 
```

Comment:
@CjMoore are we keeping this a float for now? Still interested in potentially using wizard money, but then we would have to re-assess trying to do the stripe or paypal extension @meyerhoferc @valerietrudell